main_server/local_server/install_wizard/installer_ajax.php

```diff
@@ -30,6 +30,11 @@
 require_once(dirname(__FILE__).'/../../server/shared/classes/comdef_utilityclasses.inc.php');
 require_once(dirname(__FILE__).'/../../server/shared/Array2Json.php');
 
+function generateRandomString($length = 10)
```

Comment:
Used to generate random passwords. This is not really a cryptographically secure random string generator, but it's close enough for BMLT purposes.